content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -429,6 +430,58 @@ memory      0     20Gi
 pods        0     10
 ```
 
+### Cross Namespace Pod Affinity Quota
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Operators can use `CrossNamespacePodAffinity` quota scope to limit which namespaces are allowed to have pods 
+with affinity terms that cross namespaces (i.e., allowed to set `namespaces` or `namespaceSelector` fields in pod affinity terms)
+
+Preventing users from using cross-namespace affinity terms might be desired since a pod with anti-affinity constraints can block pods from all other namespaces 
+from getting scheduled in a failure domain. 
+
+Using this scope operators can prevent certain namespaces (`foo-ns` in the example below) 
+from having pods that use cross-namespace pod affinity by creating a resource quota object in
+that namespace with `CrossNamespaceAffinity` scope and hard limit of 0:
+
+```yaml
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: disable-cross-namespace-affinity
+  namespace: foo-ns
+spec:
+  hard:
+    pods: "0"
+  scopeSelector:
+    matchExpressions:
+    - scopeName: CrossNamespaceAffinity
+```
+
+If operators want to disallow using `namespaces` and `namespaceSelector` by default, and 
+only allow it for specific namespaces,  they could configure `CrossNamespaceAffinity` 
```

Comment:
Considering that this paragraph is in the "resource-quotas" page, mentioning "namespaces" and "namespaceSelector" in this context is confusing.

I'll take a look at the `ResourceQuotaConfiguration` and see if we can generate a reference API for it.

Reply:
sg, let me know if you have any suggestions.